### ISSUE TITLE
can not set MONGO_MAX_POOL_SIZE

### ISSUE THREAD

**Issue by strahe** [dcrosta/flask-pymongo#77]
When i set MONGO_MAX_POOL_SIZE = 10 ,I get a error
"pymongo.errors.ConfigurationError: Unknown option max_pool_size"

**Comment by DJO3** [comment]
Can you provide a sample of the code? I tried replicating the error in one of my personal applications and  did not receive the error. 

For reference, this is the config I used:
`application = Flask(__name__)`
`application.config["MONGO_HOST"] = "mongoContainer"`
`application.config["MONGO_DBNAME"] = "setlists"`
`application.config['SERVER_NAME'] = 'setlist.visualizer'`
`application.config['MONGO_MAX_POOL_SIZE'] = 10`
`application.config["MONGO_CONNECT"] = False`
`mongo = PyMongo(application, config_prefix='MONGO')`

**Comment by strahe** [comment]
this is my test code:
```
from flask import Flask
from flask.ext.pymongo import PyMongo
app = Flask(__name__)
class Config:
    DEBUG = True
    # MongoDB
    MONGO_HOST = '127.0.0.1'
    MONGO_PORT = 27017
    MONGO_DBNAME = 'stock'
    MONGO_USERNAME = 'stock'
    MONGO_PASSWORD = 'uDZIVYP38zo6alp'
    MONGO_SOCKET_TIMEOUT_MS = 3000
    MONGO_MAX_POOL_SIZE = 10
app.config.from_object(Config)
mg = PyMongo(app)
if __name__ == '__main__':
    app.run()
```
this is the error:
```
(.venv) [strahe@strahe stock]$ python test.py 
/home/strahe/git/stock/.venv/lib/python3.5/site-packages/flask/exthook.py:71: ExtDeprecationWarning: Importing flask.ext.pymongo is deprecated, use flask_pymongo instead.
  .format(x=modname), ExtDeprecationWarning
Traceback (most recent call last):
  File "test.py", line 23, in <module>
    mg = PyMongo(app)
  File "/home/strahe/git/stock/.venv/lib/python3.5/site-packages/flask_pymongo/__init__.py", line 97, in __init__
    self.init_app(app, config_prefix)
  File "/home/strahe/git/stock/.venv/lib/python3.5/site-packages/flask_pymongo/__init__.py", line 249, in init_app
    cx = connection_cls(*args, **kwargs)
  File "/home/strahe/git/stock/.venv/lib/python3.5/site-packages/pymongo/mongo_client.py", line 342, in __init__
    for k, v in keyword_opts.items())
  File "/home/strahe/git/stock/.venv/lib/python3.5/site-packages/pymongo/mongo_client.py", line 342, in <genexpr>
    for k, v in keyword_opts.items())
  File "/home/strahe/git/stock/.venv/lib/python3.5/site-packages/pymongo/common.py", line 471, in validate
    value = validator(option, value)
  File "/home/strahe/git/stock/.venv/lib/python3.5/site-packages/pymongo/common.py", line 107, in raise_config_error
    raise ConfigurationError("Unknown option %s" % (key,))
pymongo.errors.ConfigurationError: Unknown option max_pool_size
```
and the version Information:
```
click==6.6
Flask==0.11.1
Flask-PyMongo==0.4.1
itsdangerous==0.24
Jinja2==2.8
MarkupSafe==0.23
pymongo==3.2.2
Werkzeug==0.11.10
```

**Comment by hbldh** [comment]
I actually happened to make what I believe is a fix for this error yesterday: In PyMongo>=3.0, the parameter max_pool_size is renamed maxPoolSize, which will lead to that very problem. Made a pull request #78 with the change.

I was making some modifications to fit another issue and did this as well.

**Comment by dcrosta** [comment]
Closed by #78
